### PR TITLE
Chromium/WebExtension: Fix warning

### LINKED ISSUE #12775: Warning about missing specification of browser_style
Steps to reproduce:
1. have a fresh and clean firefox profile (firefox 55.0.3 in my case) started from command line
2. install HTTPS Everywhere (Version 2017.9.12 from AMO)
3. have a look at the output

What happens:
Every time this addon is installed or firefox is started, I get this warning:
`Please specify whether you want browser_style or not in your browser_action options.`
This warning is sometimes prefixed with `[timestamp]	addons.xpi	WARN	` and sometimes with `[timestamp]	addons.webextension.https-everywhere@eff.org	WARN	`.

What should happen:
No warning

Additional info:
`browser_action` is set in https://github.com/EFForg/https-everywhere/blob/master/chromium/manifest.json#L18. 

It doesn't look like the MDN docs have a related documentation:
https://developer.mozilla.org/en-US/Add-ons/WebExtensions/API/browserAction
https://developer.mozilla.org/en-US/Add-ons/WebExtensions/user_interface/Browser_action

### PR DESCRIPTION
**Pull request by genodeftest**
This warning (see #12775 for details) is caused by not specifying `browser_style` in the `browser_action` section.

This change has been tested on Firefox 55.0.3, but not on any other version. It is not tested on Firefox ESR or Chromium at all.
Fixes #12775

### CODE REVIEW

**Comment by J0WI** [comment]
This might be something to look at before the next release. @Hainish

**Comment by Hainish** [comment]
@bcyphers can you look at this?

**Comment by genodeftest** [comment]
I broke it now. Will re-upload a new fix soon.

File: chromium/manifest.json [hunk 1]
```diff
@@ -25,6 +25,7 @@
         },
         "default_popup": "pages/popup/index.html",
         "default_title": "__MSG_about_ext_name__"
```
[the remark this answers is not in the document]
Comment (genodeftest) [reply chromium/manifest.json]: Yep, that's what I broke.

**Comment by genodeftest** [comment]
Done, should be ready to merge now.

**Comment by genodeftest** [comment]
Is travis broken? The failures don't look like they were related to my code.

**Comment by genodeftest** [comment]
@Giltyhub : Conflict resolved.

@bcyphers : Done, comma is now added. Sorry this took me so long.

The tests fail in `test/selenium/test_options.py::OptionsTest::test_show_counter FAILED` for stable firefox and chrome beta, but I don't know whether this is related to my code changes.

**Comment by Giltyhub** [comment]
@genodeftest thanks for solving the conflict! travis ci fail most probably is https://github.com/EFForg/https-everywhere/issues/15239

**Comment by genodeftest** [comment]
@Giltyhub : Thanks for this tip. After wasting a few cpu hours the checks finally passed.

**Comment by Hainish** [comment]
@bcyphers please review

**Comment by bcyphers** [comment]
I was unable to reproduce #12775 in the latest firefox or chrome, but it looks like the bug is fixed in ff55 and everything else still works. Good to go.

**Comment by J0WI** [comment]
What about Tor/Firefox ESR?

**Comment by pipboy96** [comment]
I can no longer reproduce the warning in Chrome, FF and Tor Browser.

**Pull request closed without merging**